.circleci/validate-testsuite.sh

```diff
@@ -0,0 +1,84 @@
+set -euo pipefail
```

Comment:
Same comment - let's make sure that this is started from `generic-bash-template.sh`